.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and test
+
+on:
+  push:
```

Comment:
You should also enable it on Pull Requests.

Reply:
Doesn't that lead to duplicate runs?

Reply:
Yes. If you want to avoid them, you could disable directly pushing on `master` and enable the workflow only on Pull Requests.

Reply:
This isn't obvious to me from the docs: does the push event not trigger for all the relevant PR actions?

Reply:
Let's try out and see whether we're missing anything with just pushes.

Reply:
Seems like the CI isn't triggered on pushes to PR branches in external repos.

Reply:
I'm working on this.